ci_cd_scripts/check_availability.sh

```diff
@@ -1,56 +0,0 @@
-#!/bin/sh
-
-# This script try to connect to elastic search and access the DB
-# return 0 if success / return 1 if error
-
-ASSEMBL_ROOT=$VIRTUAL_ENV/..
-LOCAL_INI=$ASSEMBL_ROOT/local.ini
-
-elasticsearch_host="$(grep "elasticsearch_host" $LOCAL_INI | tr -d ' ' | awk -F "=" '{print $2}')"
```

Comment:
Not sure it is relevant to this PR but I will mention it anyway. Often I had to do a reindex-all-contents for elastic search on production servers. Since we are not supposed to have SSH access to production instances in the future, we won't be able to do that. just raising a flag @vincentfretin

Reply:
Please read all the PR. The reindexing is now handled if the mapping changed.